gcp-observability/src/main/java/io/grpc/gcp/observability/GcpObservability.java

```diff
@@ -45,23 +63,33 @@ public static synchronized GcpObservability grpcInit() throws IOException {
     if (instance == null) {
       GlobalLoggingTags globalLoggingTags = new GlobalLoggingTags();
       ObservabilityConfigImpl observabilityConfig = ObservabilityConfigImpl.getInstance();
-      Sink sink = new GcpLogSink(observabilityConfig.getDestinationProjectId(),
-          globalLoggingTags.getLocationTags(), globalLoggingTags.getCustomTags(),
-          observabilityConfig.getFlushMessageCount());
+      Sink sink =
+          new GcpLogSink(
+              observabilityConfig.getDestinationProjectId(),
+              globalLoggingTags.getLocationTags(),
+              globalLoggingTags.getCustomTags(),
+              observabilityConfig.getFlushMessageCount());
       LogHelper helper = new LogHelper(sink, TimeProvider.SYSTEM_TIME_PROVIDER);
       ConfigFilterHelper configFilterHelper = ConfigFilterHelper.factory(observabilityConfig);
```

Comment:
Since we are passing `observabilityConfig` to the non-static `grpcInit` we can construct `configFilterHelper` in there? Unless I am missing something?